docs/source/guide/s3.rst

```diff
@@ -37,45 +37,142 @@ config parameter when you create your client or resource.::
 Using the Transfer Manager
 --------------------------
 
-The `s3 transfer manager`_ provides you with less painful multipart uploads and
-downloads. Its functions are automatically added into the client when you create
-it, so there is no need to create your own transfer manager. Below you will see
-several examples of how to use it.
+``boto3`` provides interfaces for managing various types of transfers with
+S3. Functionality includes:
 
-The methods on the base client are :py:meth:`S3.Client.upload_file` and
-:py:meth:`S3.Client.download_file`::
+* Automatically managing multipart and non-multipart uploads
+* Automatically managing multipart and non-multipart downloads
+* Automatically managing multipart and non-multipart copies
+* Uploading from:
+
+  * a file name
+  * a readable file-like object
+
+* Downloading to:
+
+  * a file name
+  * a writeable file-like object
+
+* Tracking progress of individual transfers
+* Managing retries of transfers
+* Configuring various transfer settings such as:
+
+  * Max request concurrency
+  * Multipart transfer thresholds
+  * Multipart transfer part sizes
+  * Number of download retry attempts
+
+
+Uploads
+~~~~~~~
+The managed upload methods are exposed in both the client and resource
+interfaces of ``boto3``:
+
+* :py:class:`S3.Client` method to upload a file by name: :py:meth:`S3.Client.upload_file`
+* :py:class:`S3.Client` method to upload a readable file-like object: :py:meth:`S3.Client.upload_fileobj`
+* :py:class:`S3.Bucket` method to upload a file by name: :py:meth:`S3.Bucket.upload_file`
+* :py:class:`S3.Bucket` method to upload a readable file-like object: :py:meth:`S3.Bucket.upload_fileobj`
+* :py:class:`S3.Object` method to upload a file by name: :py:meth:`S3.Object.upload_file`
+* :py:class:`S3.Object` method to upload a readable file-like object: :py:meth:`S3.Object.upload_fileobj`
+
+.. note::
+
+   Even though there is an ``upload_file`` and ``upload_fileobj`` method for
+   a variety of classes, they all share the exact same functionality.
+   Other than conveniency, there are no benefits from using one method from
```

Comment:
conveniency -> convenience ?

I know it's a real word, but it's pretty archaic.